virtool/users/data.py

```diff
@@ -219,9 +229,19 @@ async def update(self, user_id: str, data: UpdateUserRequest):
                 permissions,
             )
 
-        user = await fetch_complete_user(self._authorization_client, self._mongo, self._pg, user_id)
+        user = await fetch_complete_user(
+            self._authorization_client, self._mongo, self._pg, user_id
+        )
 
         if user is None:
             raise ResourceNotFoundError
 
         return user
+
+    async def check_is_empty(self) -> bool:
+        """
+        Checks if the "users" collection is empty.
```

Comment:
In retrospect, this naming is not great. Should be `check_if_any_exist`.

Please don't refer to the underlying implementation (users collection; MongoDB) in the docstring. The point of the data layer is to be an abstraction over the data sources.